src/cmd/list.go

```diff
@@ -252,7 +259,23 @@ func listOutput(images, containers []map[string]interface{}) {
 			status := container[statusKey].(string)
 			imageName := container["Image"].(string)
 
-			fmt.Fprintf(writer, "%s\t%s\t%s\t%s\t%s\n", id, nameString, created, status, imageName)
+			// Prepare color for a line based on toolbox's status
+			// By default don't colorize a line
+			isRunning := false
+			if podman.CheckVersion("2.0.0") {
+				isRunning = status == "running"
+			} else {
+				isRunning, _ = regexp.MatchString("^Up [0-9]* [a-z]* ago$", status)
```

Comment:
I salute your perseverance!

If I were to nitpick, I'd suggest first compiling the regular expression separately outside the loop and then matching it. See [regexp.Compile](https://golang.org/pkg/regexp/#Compile) and [regexp.Regexp.MatchString](https://golang.org/pkg/regexp/#Regexp.MatchString).

Note that the `regexp` APIs can fail if there's a mistake in the regular expression, which would be a programmer error. That makes me think that we should use [regexp.MustCompile](https://golang.org/pkg/regexp/#MustCompile) in some place like `init`. That way, if we ever typo the regular expression, `toolbox` will panic on start-up and it will be a lot harder for us to miss it.

All that said, why not use just [strings.HasPrefix](https://golang.org/pkg/strings/#HasPrefix) and [strings.HasSuffix](https://golang.org/pkg/strings/#HasSuffix)? No need to handle any errors, and likely faster than regular expressions. :)

To be honest, I would still be very happy if we coloured the output only with Podman >= 2.0.0.

Reply:
I am going to leave this for you to decide. :)

Reply:
We can use a separate pull request for it.